composer.json

```
@@ -108,6 +108,7 @@
             "Sylius\\Bundle\\ResourceBundle\\spec\\": "src/Bundle/spec/",
             "Sylius\\Bundle\\ResourceBundle\\Tests\\": "tests/Bundle/",
             "Sylius\\Component\\Resource\\spec\\": "src/Component/spec/",
+            "Sylius\\Component\\Resource\\Tests\\": "tests/Component/",
```

```suggestion
            "Tests\\Sylius\\Bundle\\ResourceBundle\\": "tests/Bundle/",
            "Sylius\\Component\\Resource\\spec\\": "src/Component/spec/",
            "Tests\\Sylius\\Component\\Resource\\": "tests/Component/",
```
Wdyt? That's how we usually create namespaces for `tests` dir. It doesn't suggest the tests are part of the `Sylius\Component\Resource` namespace.

---

Hum, yeah, you're right, but that could be harder to review for now and resource bundle tests are with the same namespace.
Could we merge like that and making another PR with this change for component and bundles?